app/services/users/update.rb

```diff
@@ -117,7 +117,7 @@ def update_profile
     def conditionally_resave_articles
       return unless resave_articles? && !@user.suspended?
 
-      Users::ResaveArticlesWorker.perform_async(@user.id)
+      Articles::ResaveForAssociationWorker.perform_async(@user.model_name.name, @user.id)
     end
 
     def resave_articles?
```

Comment:
will organization need (in the future, out of scope for this change) something equivalent to this predicate?

Reply:
I think so